src/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--0 :nth-child(-n + 5),
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :nth-child(n + 5) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
don't see any sense in adding a no-active background modificator, you can add a no-active start for every start
after that add modificator only for the active start
for instance 

<img width="581" alt="Screenshot 2023-08-15 at 20 13 11" src="https://github.com/mate-academy/layout_stars/assets/64746943/0400e91a-cb89-49be-b865-0fe4b415216a">